src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -56,8 +56,11 @@ def load_command_table(self, _):
         g.custom_command('start', 'app_start', supports_no_wait=True)
         g.custom_command('stop', 'app_stop', supports_no_wait=True)
         g.custom_command('restart', 'app_restart', supports_no_wait=True)
-        g.custom_command('log', 'app_tail_log')
-        g.custom_command('log tail', 'app_tail_log', deprecate_info=g.deprecate(redirect='az spring-cloud app log', hide=True))
+        g.custom_command('logs', 'app_tail_log')
+
+    with self.command_group('spring-cloud app log', client_factory=cf_spring_cloud,
+                            deprecate_info=g.deprecate(redirect='az spring-cloud app logs', hide=True)) as g:

```

Comment:
Maybe you can add when it's not available any more.

Reply:
Thanks a lot for your advice. I'm afraid that we are not sure when it will be not available any more. Once we have a specific time, we will update it.